tools/run_tests/xds_k8s_test_driver/framework/infrastructure/gcp/compute.py

```diff
@@ -181,6 +181,9 @@ def create_url_map(
                 }],
             })
 
+    def create_url_map_with_content(self, url_map: Any) -> GcpResource:
```

Comment:
Head's up: I'm working on a big update for all classes in infrastructure.gcp. A lot of renames and logic changes.